api/src/pcapi/core/offerers/models.py

```diff
@@ -326,6 +326,30 @@ def field_exists_and_has_changed(self, field: str, value: typing.Any) -> typing.
             raise ValueError(f"Unknown field {field} for model {type(self)}")
         return getattr(self, field) != value
 
+    @property
+    def current_pricing_point_id(self) -> Optional[None]:
+        now = datetime.utcnow()
+        timespan = db_utils.make_timerange(start=now, end=None)
+        return (
+            db.session.query(VenuePricingPointLink.pricingPointId)
+            .filter(VenuePricingPointLink.venueId == self.id, VenuePricingPointLink.timespan.overlaps(timespan))  # type: ignore [attr-defined]
+            .order_by(sa.desc("id"))
+            .scalar()
```

Comment:
```suggestion
            .one_or_none()
```

... peut-être avec `scalar()` derrière, mais on ne devrait pas avoir besoin de `order_by()`, puisqu'il y a une contrainte en base de données qui n'autorise pas un lieu à avoir 2 lignes de VenuePricingPointLink sur 2 périodes de temps qui se chevauchent.

Reply:
Bien vu pour la contrainte, cet `order_by` n'avait aucun sens :)
[scalar()](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.scalar) fait déjà le nécessaire: 
```
Return the first element of the first result or None if no rows present. If multiple rows are returned, raises MultipleResultsFound.
```